### PR TITLE
Updating a filter now sends an 'element.update' notification

### PR DESCRIPTION
**Pull request by archendel**
Signed-off-by: LE SAULNIER Kevin <kevin.lesaulnier@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/filter/server/NotificationService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.gridsuite.filter.server;
```
Comment (niconoir) [inline src/main/java/org/gridsuite/filter/server/NotificationService.java:1]: Missing header

File: src/main/java/org/gridsuite/filter/server/NotificationService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Service;
+
+@Service
```
Comment (niconoir) [inline src/main/java/org/gridsuite/filter/server/NotificationService.java:14]: Missing author

**Review by niconoir — COMMENTED** [review]
Just missing headers

File: src/main/java/org/gridsuite/filter/server/FilterService.java [hunk 3]
```diff
@@ -153,6 +157,7 @@ public <F extends AbstractFilter> void changeFilter(UUID id, F newFilter) {
         } else {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, FILTER_LIST + id + NOT_FOUND);
         }
+        notificationService.emitElementUpdated(id, userId);
```
Comment (Meklo) [inline src/main/java/org/gridsuite/filter/server/FilterService.java:160]: If the element to update is not found the server still sends a notification ?

**Review by Meklo — APPROVED** [review]

**Review by niconoir — APPROVED** [review]

**Pull request merged**